src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_AlloyBlastSmelter.java

```diff
@@ -255,6 +261,11 @@ public boolean checkRecipe(final ItemStack aStack) {
 							}
 						}
 					}
+					tInputs = Arrays.copyOfRange(tInputList.toArray(new ItemStack[tInputList.size()]), 0, tInputList.size());
```

Comment:
why copy the array? i don't think it will cause problem if you directly it.

Reply:
This was how the original code did it.